src_c/camera_v4l2.c

```diff
@@ -29,42 +29,64 @@ char **
 v4l2_list_cameras(int *num_devices)
 {
     char **devices;
-    char *device;
+    char *device = NULL;
     int num, i, fd;
 
     num = *num_devices;
```

Comment:
why is num not initialized to 0 ? I mean later it is used as an index into the  devices array and incremented, therefore if should at most have the value of 64 I guess.

Reply:
`num` is assigned to `*num_devices`, and the caller has already set `*num_devices` to 0 :)

Reply:
correction: at most 65

Reply:
Well I see your point, the current code is fine but it relies on the caller `0` initialising the argument passed by reference which is not ideal, more explicit initialisation to 0 is better here

Reply:
good to go I think now